.github/workflows/main.yaml

```diff
@@ -57,6 +57,7 @@ jobs:
     - uses: actions/checkout@v4
       with:
         fetch-depth: 0
+        lfs: true
```

Comment:
I think this should also be set in the publish workflow.

Reply:
Thx good catch